sunpy/map/tests/test_mapbase.py

```diff
@@ -280,6 +288,15 @@ def test_world_to_pixel(generic_map):
     assert_quantity_allclose(test_pixel, generic_map.reference_pixel)
 
 
+def test_save(generic_map):
+        # Test save out
+        eitmap = sunpy.map.Map(a_fname)
+        afilename = tempfile.NamedTemporaryFile(suffix='fits').name
+        eitmap.save(afilename, filetype='fits', clobber=True)
+        backin = sunpy.map.Map(afilename)
+        assert isinstance(backin, sunpy.map.sources.EITMap)
```

Comment:
adding some test to make sure the `data` and `meta` attributes survied the trip would be very valuable.

Reply:
Like this?